xml/hwbp_registers.xml

```diff
@@ -766,6 +775,9 @@
             \Rxepc or \RcsrDpc (depending on \FcsrMcontrolSixAction) must be set
             to the virtual address of the next instruction that must be executed
             to preserve the program flow.
+
+            If the instruction performed multiple memory accesses, all of them
+            have been completed.
```

Comment:
For completeness, should we also say the same thing on "after"?

Reply:
I decided not to because I don't see how you can have "at least one additional instruction retired" without having all these memory accesses be complete. Is that allowed by RISC-V somewhere?

If you feel strongly about this I'll add it.

Reply:
Good point.  Never mind.